cpp/ql/src/semmle/code/cpp/Type.qll

```diff
@@ -1243,7 +1245,12 @@ class TemplateParameter extends UserType {
   override predicate involvesTemplateParameter() { any() }
 }
 
-/** A C++ template template parameter, e.g. template &lt;template &lt;typename,typename> class T>. */
+/**
+ * A C++ template template parameter, for example:
+ * ```
+ * template <template <typename,typename> class T>
+ * ```
+ */
```

Comment:
I wonder if we should not use an example that motivates the use of template template parameters.  Here's what I found on the internet (and performed renaming):
```
template <template<typename> class Container, class Element>
void First(const Container<Element> &container) { }
```
What do you think?